xcode/Subconscious/Shared/Components/Common/SubtextView.swift

```diff
@@ -91,7 +92,8 @@ struct SubtextView: View {
             guard let link = url.toSubSlashlinkLink()?.toEntryLink() else {
                 return .systemAction
             }
-            onLink(link)
+            
+            onLink(link.rebaseIfNeeded(peer: peer))
```

Comment:
Rebasing link taps but not embedded transclusions seems conceptually surprising (see https://github.com/subconsciousnetwork/subconscious/pull/1040/files#r1432707403). Can you help walk me through what is going on here?

Reply:
The short version is that when we use a `TranscludeView` we pass it an `EntryStub` which comes from the local SQLite index (as a result of fetching transcludes). The address for this `EntryStub` is already fully resolved, or else we wouldn't present the transclude in the first place (because it wouldn't be in the index).

We _could_ work around this by intentionally ignoring the address on the `EntryStub` and deriving it another way, but that is just wasted work ultimately.

Links are different, until we tap a link we do not always know what's on the other end (or else we would typically present a transclude).

Reply:
^ not to say any of this _has to_ or _must_ be this way, just describing the status quo and why transcludes work differently.